.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# global owners for nimi-python
```

Comment:
This comment doesn't add much value. You can get rid of it.

Reply:
done :)